README.md

```diff
@@ -23,6 +23,15 @@ Clojure PDF manipulation library & wrapper for [PDFBox](http://pdfbox.apache.org
 (pdf/merge-pdfs :input ["pdfs/clojure-1.pdf" "pdfs/clojure-2.pdf"] :output "foo.pdf")
 ```
 
+### Split a PDF into mutliple PDDocuments
+```clojure
+ (require '[pdfboxing.split :as pdf])
```

Comment:
Thank you for writing the documentation, much appreciated!

One thing I'd ask is if maybe instead of having the comments inline like you've added them, having them on their own above the code sample, e.g:


### Split a PDF into mutliple PDDocuments

```clojure 
(require '[pdfboxing.split :as pdf])
```

List of PDDocument pages 1 through 8

```clojure
(pdf/split-pdf :input "pdfs/clojure.pdf" :start 1 :end 8)
```
Splits the pdf into single pages as a list of PDDocument

```clojure
(pdf/split-pdf :input "pdfs/clojure.pdf") 
```

Splits the pdf in half and writes them to disk as clojure-1.pdf and clojure-2.pdf

```clojure
(pdf/split-pdf-at :input "pdfs/clojure.pdf") 
```

Splits into two pdfs the first having 5 pages, second has rest
```clojure
(pdf/split-pdf-at :input "pdfs/clojure.pdf" :split 5) 
```